_config.yml

```diff
@@ -23,11 +23,21 @@ defaults:
       githubbranch: "v1.7.3"
       docsbranch: "release-1.7"
       versions:
+      - fullversion: "v1.11.0"
+          version: "v1.11"
+          githubbranch: "v1.11.0"
+          docsbranch: "release-1.11"
+          url:  https://kubernetes.io/
```

Comment:
Please consider use 'https://kubernetes.io/docs' as the URL. Using the "/" URL path makes me nervous.

Reply:
Done @tengqm